test/system/static.test.ts

```diff
@@ -185,6 +185,33 @@ test("static analysis provides hashes for key binaries", async (t) => {
   );
 });
 
+test("static analysis provides hashes for found openjdk binaries", async (t) => {
+  const thisIsJustAnImageIdentifierInStaticAnalysis = "openjdk:doesnotexist";
+  const dockerfile = undefined;
+  const pluginOptions = {
+    staticAnalysisOptions: {
+      imagePath: getFixture("docker-save/openjdk.tar"),
+      imageType: ImageType.DockerArchive,
+    },
+  };
+
+  const pluginResult = (await plugin.inspect(
+    thisIsJustAnImageIdentifierInStaticAnalysis,
+    dockerfile,
+    pluginOptions,
+  )) as PluginResponseStatic;
```

Comment:
not sure we need the ```as PluginResponseStatic``` here

Reply:
looks like we do as otherwise it treats it as 'PluginResponse' and in that case it doesn't recognise the hashes